src/org/openlcb/swing/memconfig/MemConfigDescriptionPane.java

```diff
@@ -50,6 +53,7 @@ public MemConfigDescriptionPane(NodeID node, MimicNodeStore store,
         setLayout(new BoxLayout(this, BoxLayout.Y_AXIS));
 
         addLine(commandLabel, "Commands:");
+        addLine(writesLabel, "Write Codes:");
```

Comment:
How about "Writes Supported:" or "Write Modes:"

Reply:
Change to Write Modes, thanks for suggestion.